Makefile

```diff
@@ -17,7 +17,7 @@ ifeq ($(OS),Darwin)
 endif
 
 ifeq ($(OS),Linux)
-	LD_LIBRARY_PATH += $(CURDIR)/operator/risc_zero/lib
+	LD_LIBRARY_PATH+=$(CURDIR)/operator/risc_zero_old/lib:$(CURDIR)/operator/risc_zero/lib
```

Comment:
Now that I pay more attention to this, the correct pattern isn't this here, it's:
```suggestion
	LD_LIBRARY_PATH=$(CURDIR)/operator/risc_zero_old/lib:$(CURDIR)/operator/risc_zero/lib:$(LD_LIBRARY_PATH)
```

Reply:
That is, you do not append a string (it will be separated by a space, while lookup paths are separated by colon), but instead prepend and separate with `:` from the originals.